lib/cuke_linter/linters/background_does_more_than_setup_linter.rb

```diff
@@ -4,17 +4,33 @@ module CukeLinter
 
   class BackgroundDoesMoreThanSetupLinter < Linter
 
+    # Changes the linting settings on the linter using the provided configuration
+    def configure(options)
+      @when_keywords = DialectHelper.get_configured_keywords(options, DEFAULT_WHEN_KEYWORD)
+      @then_keywords = DialectHelper.get_configured_keywords(options, DEFAULT_THEN_KEYWORD)
+    end
+
     # The rule used to determine if a model has a problem
     def rule(model)
       return false unless model.is_a?(CukeModeler::Background)
 
-      model.steps.collect(&:keyword).any? { |keyword| (keyword == 'When') || (keyword == 'Then') }
+      model.steps.collect(&:keyword).any? { |keyword| when_keywords.include?(keyword) || then_keywords.include?(keyword) }
     end
 
     # The message used to describe the problem that has been found
     def message
       'Background has non-setup steps'
     end
 
+    private
+
+    def when_keywords
+      @when_keywords || [DEFAULT_WHEN_KEYWORD]
+    end
+
+    def then_keywords
+      @then_keywords || [DEFAULT_THEN_KEYWORD]
+    end
+
```

Comment:
...and all of this can go away because when the linter was configured it was _completely_ configured.

Follow the same pattern with other linters as needed and it'll be about as clean as it's going to get. :)

Reply:
done